build

```diff
@@ -5,12 +5,21 @@ if [ -z "$PLRUST_TARGET" ]; then
     export PLRUST_TARGET="x86_64-unknown-linux-postgres"
 fi
 
+# Make sure the tip of pgx's develop branch is used,
+# until a release that has all the necessary features is cut.
+cargo update -p pgx
+cargo fetch
+cargo install cargo-pgx --git "https://github.com/tcdi/pgx.git" --branch "develop"
+# Don't need to run cargo pgx init: user might already have set it up,
+# and doing so risks clobbering their configuration.
+# If they get an error, it's fairly self-explanatory.
+
 (
     if cd postgrestd; then
         git pull
         git submodule update --init --recursive
     else
-        git clone https://github.com/tcdi/postgrestd.git --recurse-submodules
+        git clone https://github.com/tcdi/postgrestd.git --recurse-submodules --branch "cargo-lock"
```

Comment:
Is the `cargo-lock` branch temporary to get a passing result in CI before making it official?

Reply:
Yeah, was just checking. Gonna revert this immediately basically.